chrome/locale/en-US/quicktext.properties

```diff
@@ -11,7 +11,7 @@ template=Template
 newGroup=New group
 newTemplate=New template
 newScript=New script
-remove=Are you sure you want to remove "%S"?
+remove=Are you sure you want to remove “%S”?
```

Comment:
you use differnt quotes for differnt languages (sometimes at the top, sometimes atthe bottom). Where is that defined?

Reply:
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table